README.md

```diff
@@ -1,2 +1,29 @@
-# index-mermaid-graph
-a tool that produces SVG diagrams using OLM graphs as the input
+## Output Mermaid graph script from Operator Lifecycle Manager indices
+
+### Required:
+make, sqlite3, go v14+, container execution environment (such as Docker, podman)
```

Comment:
Could we add here the requirements with bollet points.
Also, I do not think that we need to add make here. 

Then, what do you mean with is required `sqlite3`. Is not that an go dep? Should I need to install something locally to run the project? 

Are you sure that the projects works with docker/podman and all? Is not better we stick here with docker only for example?

Reply:
Cleaned it up to a bulleted list
`make` is definitely a requirement and people may have to go out and get it.
`sqlite3` is a requirement to be installed by the user and not a Go dep. This decouples the visualizer tool from sqlite3 and means that it should only require minimal changes, possibly just a JSON pre-processor outside of the Go code, to work with declarative config down the road.
Yes, tested with both Docker and Podman.